Gulpfile.js

```diff
@@ -0,0 +1,52 @@
+var gulp = require('gulp');
+var uncss = require('gulp-uncss');
+
+gulp.task('uncss', function() {
+  return gulp.src([
+      '_site/lib/material-design-lite/material.min.css',
+      '_site/lib/foundation/css/foundation.css'
+    ])
+    .pipe(uncss({
+      html: [
+        "http://localhost:4000/404.html",
```

Comment:
Is there anyway to dynamically generate this list?

currently someone would have to update it every time we create a new page.